### PR TITLE
Autogenerate surface operator docs 

### PR DESCRIPTION
**Pull request by MingweiSamuel**
Moves each operators into its own file

### CODE REVIEW

File: hydroflow/src/lib.rs [hunk 1]
```diff
@@ -46,4 +46,6 @@ mod booktest {
     booktest!(surface_flows);
```
Comment (jhellerstein) [inline hydroflow/src/lib.rs:46]: what's with the empty chat_driver.py file above?

**Review by jhellerstein — APPROVED** [review]

File: hydroflow_internalmacro/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[package]
+name = "hydroflow_internalmacro"
```
[the remark this answers is not in the document]
Comment (MingweiSamuel) [reply hydroflow_internalmacro/Cargo.toml:2]: looks like only the `rwx` permissions changed so github doesnt show the file content, Ill see if I can undo that change

File: hydroflow/src/lib.rs (hunk 1 above)
Comment (MingweiSamuel) [reply hydroflow/src/lib.rs:46]: `rwx` permissions changed only, will try to remove the change

File: book/SUMMARY.md [hunk 3]
```diff
@@ -18,5 +18,34 @@
   - [Flows](./surface_flows.md)
   - [Data Sources](./surface_data.md)
   - [Operators](./surface_ops.md)
+  - [Operators GEN](./surface_ops.gen.md)
+    <!-- AUTOGENERATED SECTION START -->
```
[the remark this answers is not in the document]
Comment (MingweiSamuel) [reply book/SUMMARY.md:22]: Fixed after I move op docs into a single file

**Pull request merged**